### PR TITLE
Drop unused variables in Python API

### LINKED ISSUE #124: pilz_robot_programming has a lot of unused code
### Commit
commit 685c781874d1771b67058b040983813d9185a719

### Steps to reproduce
vulture .../pilz_robot_programming

### Expected behavior
Not much unused stuff

### Observed behavior
quite the opposite. And it doesn't even find all. e.g. _MAX_ACC_SCALE

### PR DESCRIPTION
**Pull request by jschleicher**
Fixes #124.

Maximum acceleration  is unused but probably should be re-considered when increasing the version (allowing breaking changes). Drop RobotMoveInvalidState exception and other unused imports.

### CODE REVIEW

**Comment by rfeistenauer** [comment]
removed the unused imports

File: pilz_robot_programming/src/pilz_robot_programming/robot.py [hunk 1]
```diff
@@ -21,14 +21,14 @@
...
-from moveit_msgs.msg import MoveItErrorCodes, MoveGroupAction
+from moveit_msgs.msg import MoveItErrorCodes
 import time
 import threading
 from std_srvs.srv import Trigger
 import tf
 import psutil
 
-from .move_control_request import _MoveControlState, MoveControlAction,_MoveControlStateMachine
+from .move_control_request import _MoveControlState, MoveControlAction, MoveControlStateMachine
```
Comment (jschleicher) [inline pilz_robot_programming/src/pilz_robot_programming/robot.py]: why did you remove the underscore?

**Review by jschleicher — COMMENTED** [review]
@rfeistenauer thanks. I'm just fixing-up the tests to run full-speed again...

File: pilz_robot_programming/src/pilz_robot_programming/robot.py (hunk 1 above)
Comment (rfeistenauer) [reply pilz_robot_programming/src/pilz_robot_programming/robot.py]: If it was me then maybe on accident while deleting the unused modules?

**Review by rfeistenauer — APPROVED** [review]
Despite the fact, that the test changes show, that there maybe should be a general speed option, everything seems to be ok.

**Comment by martiniil** [comment]
@jschleicher Changing the default acceleration changes the robots behaviour on existing programs. Why do you think it was a typo?

**Comment by jschleicher** [comment]
@martiniil I think, the PTP init line should read default=default to be consistent with Lin and Circ constructors:
https://github.com/PilzDE/pilz_industrial_motion/blob/47dfa2561c14b5b187eadb207d977a93236b782e/pilz_robot_programming/src/pilz_robot_programming/commands.py#L368
https://github.com/PilzDE/pilz_industrial_motion/blob/47dfa2561c14b5b187eadb207d977a93236b782e/pilz_robot_programming/src/pilz_robot_programming/commands.py#L429

and with the default acceleration in the base class
https://github.com/PilzDE/pilz_industrial_motion/blob/47dfa2561c14b5b187eadb207d977a93236b782e/pilz_robot_programming/src/pilz_robot_programming/commands.py#L174-L176

**Comment by jschleicher** [comment]
Rebased onto current melodic_devel to resolve merge-conflicts and squashed into a single commit.

**Comment by martiniil** [comment]
@jschleicher Reducing the default velocity requires a version increment. Cannot merge this at the moment.

**Comment by hslusarek** [comment]
> See #124.
> 
> The full velocity for PTP commands was a typo?! Maximum velocity and acceleration are then unused and can be dropped. RobotMoveInvalidState exception as well.

@jschleicher @martiniil I think a higher default velocity  for the PTP was not a typo. In my opinion we want the commands to be executed as fast as possible. However, in case of lin, circ and blending a high default velocity was not a good choice because many motions could not be performed without explicitly changing/reducing the velocity. Therefore, the default velocity was reduced. I think the general behavior should be decided by @dbakovic

File: pilz_robot_programming/src/pilz_robot_programming/commands.py [hunk 2]
```diff
@@ -320,7 +317,7 @@ class Ptp(_BaseCmd):
 
             acc_scale = vel_scale * vel_scale
     """
-    def __init__(self, vel_scale=_MAX_VEL_SCALE, acc_scale=None, *args, **kwargs):
+    def __init__(self, vel_scale=_DEFAULT_VEL_SCALE, acc_scale=None, *args, **kwargs):
```
Comment (hslusarek) [inline pilz_robot_programming/src/pilz_robot_programming/commands.py]: Please do not change to such a low default velocity. Choose a "reasonable" default velocity for PTP. This low default value was mainly intended for lin, circ and blending.

**Review by hslusarek — DISMISSED** [review]

File: pilz_robot_programming/src/pilz_robot_programming/commands.py (hunk 2 above)
Comment (ct2034) [reply pilz_robot_programming/src/pilz_robot_programming/commands.py]: @jschleicher

Comment (jschleicher) [reply pilz_robot_programming/src/pilz_robot_programming/commands.py]: Renamed the variable without changing its behaviour. Having different scales still seems like a typo to me; resolved into a new issue (related to operation modes?).

Comment (jschleicher) [reply pilz_robot_programming/src/pilz_robot_programming/commands.py]: Resolved into #191.

**Comment by jschleicher** [comment]
1422a95  reduces this PR to only drop unused variables. @ct2034 Please re-review.

Leaving default speed as open issue #191.

**Comment by jschleicher** [comment]
@ct2034 friendly ping

**Review by ct2034 — APPROVED** [review]
These changes make sense to me
